jetbrains-core/src/software/aws/toolkits/jetbrains/core/gettingstarted/OuterPanel.kt

```diff
@@ -0,0 +1,423 @@
+
+package software.aws.toolkits.jetbrains.core.gettingstarted
+
+import com.intellij.ide.ui.laf.darcula.ui.DarculaButtonUI
+import com.intellij.openapi.actionSystem.ActionManager
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.actionSystem.PlatformDataKeys
+import com.intellij.openapi.application.runInEdt
+import com.intellij.openapi.project.DefaultProjectFactory
+import com.intellij.openapi.project.DumbAwareAction
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.ui.DialogPanel
+import com.intellij.openapi.ui.DialogWrapper
+import com.intellij.ui.IdeBorderFactory
+import com.intellij.ui.JBColor
+import com.intellij.ui.dsl.builder.TopGap
+import com.intellij.ui.dsl.builder.panel
+import com.intellij.util.ui.JBFont
+import com.intellij.util.ui.components.BorderLayoutPanel
+import icons.AwsIcons
+import software.aws.toolkits.jetbrains.services.caws.CawsEndpoints
+import software.aws.toolkits.jetbrains.services.codewhisperer.util.CodeWhispererColorUtil
+import software.aws.toolkits.jetbrains.services.codewhisperer.util.CodeWhispererConstants.CODEWHISPERER_LEARN_MORE_URI
+import software.aws.toolkits.jetbrains.ui.feedback.FeedbackDialog
+import software.aws.toolkits.resources.message
+import java.awt.Color
+import java.awt.Dimension
+import javax.swing.BorderFactory
+import javax.swing.JComponent
+
+open class OuterPanel(
+    private val project: Project
+) : DialogWrapper(project) {
+
+    private val panel: DialogPanel by lazy { createPanel() }
+    init {
+
+        init()
+    }
+
+    fun createPanel() = panel {
+        row {
+            apply {
+                icon(AwsIcons.Logos.AWS_SMILE_LARGE)
+            }
+            panel {
+                row {
+                    label(message("aws.onboarding.getstarted.panel.title")).applyToComponent {
+                        font = JBFont.h1().asBold()
+                        foreground = Paneltext.TITLE_TEXT_FONTCOLOR
+                    }
+                }
+                row {
+                    text(message("aws.onboarding.getstarted.panel.comment_link_doc"))
+                    text(message("aws.onboarding.getstarted.panel.comment_link_github"))
+                    text(message("aws.onboarding.getstarted.panel.share_feedback")) { hyperlinkEvent ->
+                        close(OK_EXIT_CODE)
+                        val actionEvent = AnActionEvent.createFromInputEvent(
+                            hyperlinkEvent.inputEvent,
+                            Paneltext.SHARE_FEEDBACK_LINK,
+                            null
+                        ) { if (PlatformDataKeys.PROJECT.`is`(it)) project else null }
+                        ActionManager.getInstance().getAction("aws.toolkit.getstarted.shareFeedback").actionPerformed(actionEvent)
+                    }
+                }
+            }
+        }
+
+        group(message("aws.onboarding.getstarted.panel.group_title", "#FFFFFF")) {
```

Comment:
`TITLE_TEXT_FONTCOLOR`?

Reply:
Set as a font color for all the titles in the panel